charts/comet-admin-v5/values.yaml

```diff
@@ -1,6 +1,7 @@
-nodeEnv: production
-npmRun: serve
-timeZone: Europe/Vienna
+global:
+  nodeEnv: production
+  npmRun: serve
+  timeZone: Europe/Vienna
```

Comment:
I'm not sure if it is a good idea to set global values from a library chart? Is this even possible?

Reply:
We call them as `Values.global.nodeEnv` and the linter doesn't like it, if there is no default value

Reply:
I removed the global setting. Let's see if the linter complains